### PR TITLE
HAWQ-1011. Check whether the table to be registered is existed.

### PR DESCRIPTION
**Pull request by xunzhang**

### CODE REVIEW

**Comment by xunzhang** [comment]
cc @ictmalili @huor

File: tools/bin/hawqregister [hunk 1]
```diff
@@ -131,6 +130,18 @@ def option_parser_yml(yml_file):
 
 def create_table(dburl, tablename, schema_info, fmt, distrbution_policy, file_locations):
     try:
+        query = "select count(*) from pg_class where relname = '%s'" % tablename.split('.')[-1]
```
Comment (ictmalili) [inline tools/bin/hawqregister]: Maybe we need to consider the upper/lower cases of tablename.  All the table name stored in HAWQ are converted to lower case. So I think we need to convert tablename to lower case and then do comparison.

**Comment by xunzhang** [comment]
also cc @radarwave

**Comment by radarwave** [comment]
+1

**Comment by ictmalili** [comment]
LGTM. +1

**Comment by xunzhang** [comment]
Merged into master.

**Pull request closed without merging**